src/components/AddressBar.tsx

```diff
@@ -1,46 +1,13 @@
 import * as React from "react";
-import {useContext, useState} from "react";
+import {useContext} from "react";
 import {GlobalContext} from "../context/GlobalState";
-import {Action, ActionTypes} from "../context/AppReducer";
-import TransportWebHID from "@ledgerhq/hw-transport-webhid";
-import Tezos from "@ledgerhq/hw-app-tezos";
 
 export function AddressBar() {
-    const {globalState, dispatch } = useContext(GlobalContext);
```

Comment:
Why did you strip out all this functionality from this component? The ticket is asking you to modify only the Settings component.